self-tests.yml

```diff
@@ -652,25 +652,6 @@ buildvariants:
       - name: "js-test"
       - name: "verify-agent-version-update"
       - name: "docker-cleanup"
-      - name: test-cloud
```

Comment:
Now that docker is gone, we don't need the hack to ensure test-cloud didn't run in docker